docs/source/tutorials.rst

```diff
@@ -451,44 +451,10 @@ BTK supports galaxy image generation with galsim_hub ; please refer to :ref:`thi
 
 You can find a notebook version of this tutorial in the notebooks folder.
 
-First, you should use the ``CosmosCatalog`` catalog instead of the Catsim one. While galsim_hub only require parameters for the image generation, we have chosen to use COSMOS as the source of those parameters so as to get a realistic distribution of those parameters. We have included a small sample of the catalog in BTK, and advise you to download the full catalog (see:ref:`COSMOS`) for better results.
+.. Add notes on how to install galsim hub with pip extras
 
-.. jupyter-execute::
-
-  COSMOS_CATALOG_PATHS = [
```

Comment:
I see you are removing this from the documentation's tutorial, are you planning to put it back later (maybe as a separate  section in the documentation?)

Reply:
I feel that's nice to have it in the documentation because if it's just a separate jupyter notebook it's easy to forget about it and become outdated

Reply:
No, since it is the galsim_hub one which we agreed we don't want to run everytime.

Reply:
oh whoops I didn't mean the galsim_hub one, but rather the advanced-features one